shotover-proxy/tests/transforms/tee.rs

```diff
@@ -0,0 +1,147 @@
+use crate::helpers::ShotoverManager;
+use serial_test::serial;
+use test_helpers::docker_compose::DockerCompose;
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_ignore() {
+    ignore_matches().await;
```

Comment:
Why not just make these all invidual tests rather then batching them into groups of two?

Reply:
True, I originally had `DockerCompose` so it made sense to group them. I'll fix that up